### ISSUE TITLE
IRC-like slash commands

### ISSUE THREAD

**Issue by TuxOtaku** [sdelements/lets-chat#182]
might be nice to follow Slack's suit and include IRC-like slash commands (/me, /msg, /nick, etc.)

**Comment by hhaidar** [comment]
+1

**Comment by tejasmanohar** [comment]
```
/join #...
/msg @
/whois @ 
```

**Comment by Avalarion** [comment]
An easy usable api or plugin endpoint would also be good here.

**Comment by funkaoshi** [comment]
There is already an API for the Let's Chat. You can read about it in our documentation. Or do you mean an API for future IRC commands?

**Comment by Avalarion** [comment]
Yes, i meant for this commands to add or remove them and perhaps even handle them on a remote place.

**Comment by kevindelaney** [comment]
+1 for /me

**Comment by geoffwhittington** [comment]
This is sort of a different feature....

It would be cool if plugins could expose their own slash commands and take input from the message entry window. If Lets Chat could also expose plugin preferences to users then slash actions would be done in the context of different identities - not just one Hubot.

**Comment by marzolfb** [comment]
+1
